sdk/src/Services/MobileAnalytics/Custom/_unity/BackgroundRunner.unity.cs

```diff
@@ -104,6 +104,10 @@ private void DoWork()
                     }
                     Thread.Sleep(BackgroundSubmissionWaitTime * 1000);
                 }
+                catch(ThreadAbortException)
+                {
+                  // handle thread aports more greacefully in unity
```

Comment:
Hi. The point of this PR is to handle the exception more gracefully, and in Unity-land that is to not spew an error...which tens to end up in some crash reporting system. How do you feel about using a warning?